sw/device/lib/testing/test_status.h

```diff
@@ -0,0 +1,63 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_TEST_STATUS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_TEST_STATUS_H_
+
+#include <stdnoreturn.h>
+
+/**
+ * Indicates the status of the software running on the CPU, from a testing
+ * perspective.
+ *
+ * It is useful to track the current status of the test SW executing on the CPU
+ * as a signal the helps aid in debug.
+ *
+ * The values set to these literals is arbitrary and do not possess any special
+ * meaning (other than being hex 'words'). They are chosen to be 16-bit only.
```

Comment:
What value does this have? In general, we've been avoiding assigning special values to enum variants where possible.

Reply:
Just being extra cautious - very specific 16-bit values are harder to mimic by other entities in the testbench even by accident (for example the testbench can backdoor read / write to the RAM).

Reply:
Yup, sounds good to me.